homeassistant/components/fibaro/config_flow.py

```diff
@@ -54,6 +55,25 @@ async def _validate_input(hass: HomeAssistant, data: dict[str, Any]) -> dict[str
     }
 
 
+def _normalize_url(data: dict[str, Any]) -> dict[str, Any]:
```

Comment:
This smells like it should be of the signature
```suggestion
def _normalize_url(url: str) -> str:
```
for e.g. better testability.

Reply:
Yes that makes sense, I changed the method signature accordingly.